src/pages/Strategies/components/StrategyPageInsights/StrategyPageInsights.tsx

```diff
@@ -0,0 +1,50 @@
+import { displayMonetaryAmount, displayMonetaryAmountInUSDFormat, formatPercentage } from '@/common/utils/utils';
+import { Card, Dl, DlGroup } from '@/component-library';
+import { StyledDd, StyledDt } from '@/component-library/Text/Dl/Dl.style';
+import { useGetPrices } from '@/hooks/api/use-get-prices';
+import { STRATEGIES, StrategyType } from '@/types/strategies';
+import { getTokenPrice } from '@/utils/helpers/prices';
+
+import { useGetStrategyInsights } from '../../hooks/use-get-strategy-insights';
+
+type Props = {
+  strategyType: StrategyType;
+};
+
+const StrategyPageInsights = ({ strategyType }: Props): JSX.Element => {
+  const prices = useGetPrices();
+  const { depositedAmount, interest, earnedAmount } = useGetStrategyInsights(strategyType);
+
+  const currency = STRATEGIES[strategyType].currency;
+  const price = getTokenPrice(prices, currency.ticker);
+  return (
```

Comment:
You should not import these styles

Reply:
I'm not sure what styles do you mean here.

Reply:
Sorry, I meant that using `Dd` and `Dt` should be enought. I think you mistakenly imported the styles.